.github/workflows/deploy-formbricks-cloud.yml

```
@@ -23,6 +28,11 @@ on:
         required: false
         type: string
         default: 'ghcr.io/formbricks/formbricks'
+      ENVIRONMENT:
+        description: 'The environment to deploy to'
+        required: true
+        type: string
+        default: "stage"
```

actionlint (1.7.4)

input "ENVIRONMENT" of workflow_call event has the default value "stage", but it is also required. if an input is marked as required, its default value will never be used

(events)